include/deal.II/lac/precondition.h

```diff
@@ -1398,6 +1398,17 @@ namespace internal
 
       const VECTOR &diagonal_vector;
     };
+
+    struct EigenvalueTracker
+    {
+    public:
+      void connect(const std::vector<double> &eigenvalues)
```

Comment:
Call the function `slot()` instead. The common use of names is to *connect* a *slot* to a *signal*.